core/domain/role_services.py

```diff
@@ -157,6 +161,9 @@
         ACTION_SEND_MODERATOR_EMAILS,
         ACTION_UNPUBLISH_ANY_PUBLIC_ACTIVITY,
     ],
+    feconf.ROLE_ID_TOPIC_MANAGER: [
```

Comment:
I think you can also allow admins to edit any topic (this is probably a new action).

Reply:
Ok, I'll add this :+1:

Reply:
Done